### ISSUE TITLE
Recurring error

### ISSUE THREAD

**Issue by awesomecuber** [Monacraft/AnkiDiscord#11]
Nothing bad seems to happen, but I get this error fairly often.

```Debug info:
Anki 2.1.11 (3cf770c7) Python 3.6.7 Qt 5.12.1 PyQt 5.11.3
Platform: Windows 10
Flags: frz=True ao=True sv=2

Caught exception:
  File "aqt\webview.py", line 27, in cmd
  File "aqt\webview.py", line 85, in _onCmd
  File "aqt\webview.py", line 360, in _onBridgeCmd
  File "aqt\toolbar.py", line 56, in _linkHandler
  File "aqt\toolbar.py", line 74, in _browseLinkHandler
  File "aqt\main.py", line 833, in onBrowse
  File "aqt\__init__.py", line 82, in open
  File "<decorator-gen-8>", line 2, in __init__
  File "anki\hooks.py", line 74, in decorator_wrapper
  File "anki\hooks.py", line 65, in repl
  File "aqt\browser.py", line 404, in __init__
  File "aqt\browser.py", line 467, in setupMenus
  File "anki\hooks.py", line 29, in runHook
  File "C:\Users\Aweso\AppData\Roaming\Anki2\addons21\1133851639\__init__.py", line 113, in onBrowse
    onState("browse", "dummy")
  File "C:\Users\Aweso\AppData\Roaming\Anki2\addons21\1133851639\__init__.py", line 103, in onState
    update(dueMessage, "Browsing decks", "search")
  File "C:\Users\Aweso\AppData\Roaming\Anki2\addons21\1133851639\__init__.py", line 49, in update
    rpc_obj.set_activity(activity)
  File "C:\Users\Aweso\AppData\Roaming\Anki2\addons21\1133851639\rpc.py", line 138, in set_activity
    self.send(data)
  File "C:\Users\Aweso\AppData\Roaming\Anki2\addons21\1133851639\rpc.py", line 116, in send
    self._write(header)
  File "C:\Users\Aweso\AppData\Roaming\Anki2\addons21\1133851639\rpc.py", line 161, in _write
    self._f.flush()
<class 'OSError'>: [Errno 22] Invalid argument

**Comment by Monacraft** [comment]
Duplicate of #4

I was able to replicate this issue by restarting discord whilst having Anki open. It seems to be due to the portable RPC client this project uses. I am assuming you also got it without restarting discord?

Since it didn't cause any issues as you mentioned I did not address it at the time, but for now I can add error detection to prevent this from popping up (tho the add-on will stop displaying on discord regardless of this error catching).

If I take up this project again, I'll look into other libraries I can use for RPC in discord.
